CHANGELOG.md

```diff
@@ -1,3 +1,87 @@
+# 1.2.0 (2020-04-9)
```

Comment:
Should be `2021-04-09`. Also maybe fix the date of the rc4 release below too (should be `2021-03-01`)